template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/profilermanager/ProfilerAdminRoleConfigProviderTest.java

```diff
@@ -52,27 +54,30 @@ public void testGetRoleConfigsWithSingleRolesPerHostGroup() {
 
```

Comment:
I cannot find any testcase where SSL is enabled, and also the JDBC URL property is not checked anywhere.